src/net.h

```diff
@@ -723,7 +723,7 @@ class CNode
     {
         nRecvVersion = nVersionIn;
     }
-    int GetRecvVersion()
+    int GetRecvVersion() const
```

Comment:
That is correct, GetRefCount () is also missing 'const'.
Should the merging of this PR proceed though?